app/src/main/java/com/lastaoutdoor/lasta/ui/navigation/MainNavGraph.kt

```diff
@@ -69,14 +71,17 @@ fun NavGraphBuilder.addMainNavGraph(navController: NavHostController) {
           moreInfoScreenViewModel::changeActivityToDisplay,
           weather,
           mapState,
-          mapViewModel::updatePermission,
+          discoverScreenViewModel::updatePermission,
           initialPosition,
           initialZoom,
-          mapViewModel::updateMarkers,
-          mapViewModel::updateSelectedMarker,
-          mapViewModel::clearSelectedItinerary,
+          discoverScreenViewModel::updateMarkers,
+          discoverScreenViewModel::updateSelectedMarker,
+          discoverScreenViewModel::clearSelectedItinerary,
```

Comment:
That is a nice way for passing viewmodels functions to the screen, without having to pass the entire viewModel :)